app/code/Magento/InventoryDownloadable/Plugin/InventoryConfiguration/IsSourceItemsAllowedForProductType/DisableDownloadableTypePlugin.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryDownloadable\Plugin\InventoryConfiguration\IsSourceItemsAllowedForProductType;
+
+use Magento\Downloadable\Model\Product\Type;
+use Magento\InventoryConfiguration\Model\IsSourceItemsAllowedForProductTypeInterface;
+
+/**
+ * Disable Source items management for downloadable product type
+ */
+class DisableDownloadableTypePlugin
```

Comment:
Looks like after last discussion
We need to remove this plugin

Reply:
done